roles/core/pxe_stack/vars/CentOS_7.yml

```diff
@@ -0,0 +1,19 @@
+---
+pxe_stack_apache_cgi_module: cgi
+pxe_stack_apache_conf_path: /etc/httpd/conf.d
+pxe_stack_apache_user: apache
+pxe_stack_packages_to_install:
+  - atftp
+  - ipxe-x86_64-bluebanquise
+  - ipxe-arm64-bluebanquise
+  - httpd
+  - policycoreutils-python
+  - python36
+  - python36-PyYAML
+  - python36-clustershell
+pxe_stack_services_to_start:
+  - atftpd
+  - httpd
+pxe_stack_firewall_services_to_add:
+  - http
+  - tftp
```

Comment:
This file is similar to RedHat_7.yml which applies to CentOS_7 as well: `{{ ansible_facts.os_family }}_{{ ansible_facts.distribution_major_version }}.yml`

I would remove it.